.github/ISSUE_TEMPLATE/slack-request.md

```diff
@@ -23,7 +26,13 @@ The Slack username of the Channel/Bot/Token/Webhook owner or primary contact.
 --->
 
 
-**What Type of Request is it (Channel, Bot, Token, or Webhook)?**
+**What Type of Request is it (Channel, User Group, Bot, Token, or Webhook)?**
+<!--
+ATTENTION: Channels and UserGroup requests can now be PR'ed directly speeding up the review process. 
```

Comment:
(no strong opinions on this) do we want to stick to an issue to make sure that we have a discussion about the channel first?

Reply:
The intention is to auto-select slack-admins for review. Only a few of us watch the community repo for issues. As a PR it can be discussed there and other slack admins can review.

Reply:
just read "After you submit your request the Slack Admins will review and follow-up with any questions in the PR itself." below. I guess since we have explicitly mentioned this, it should be fine :slightly_smiling_face:

Reply:
> The intention is to auto-select slack-admins for review. Only a few of us watch the community repo for issues. As a PR it can be discussed there and other slack admins can review.


sorry, saw this comment now. Makes sense :+1: